cypress/e2e/partner-admin/default-tests/exportAllUsersFromOrg.cy.js

```diff
@@ -0,0 +1,30 @@
+const timeout = Cypress.env('timeout');
+
+const listOrgsUrl = '/list-orgs';
+
+function checkOrgExists() {
+  cy.get('ul > li', { timeout: timeout }).contains('Districts', { timeout: timeout }).click();
+
+  cy.get('div', { timeout: timeout }).should('contain.text', Cypress.env('testPartnerDistrictName'), {
+    timeout: timeout,
+  });
+  cy.log(`${Cypress.env('testPartnerDistrictName')} exists.`);
+}
+
+describe('The partner admin user', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('partnerAdminUsername'), Cypress.env('partnerAdminPassword'));
+    cy.wait(0.2 * timeout);
+    cy.navigateTo('/');
+    cy.wait(0.2 * timeout);
+    cy.navigateTo(listOrgsUrl, { timeout: timeout });
+  });
+
+  context(`when navigating to the ${Cypress.env('testPartnerDistrictName')} tab`, () => {
+    it(`should see the organization ${Cypress.env('testPartnerDistrictName')}`, () => {
+      checkOrgExists();
+      cy.get('button').contains('Export Users').click();
+      cy.readFile(`${Cypress.env('cypressDownloads')}/cypress-test-district-users-export.csv`);
```

Comment:
We should extend this test to also check the contents of the CSV as this test could still pass but result in an empty or malformed CSV. Here's an example: https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/testing-dom__csv-table/cypress/e2e/spec.cy.js

Reply:
I agree we should test this, but I think it is out of the scope for now, it should be a different unit test

Reply:
Up to you and the team to decide if you want to tackle this another time, but just saying that this E2E test will still pass after empty or malformed CSVs were generated. That's why, IMO, this should be part of this test

Reply:
Sure I am not sure how I will use that link, since we don't have a table to compare this since we don't have a table of the users we are exporting from. I will have to after export go to the users page and compare. Asking @richford  and @ksmontville to check if I should implement this.

Reply:
If the answer is yes, we should do this for all of our test that use exporting, like export score and progress reports

Reply:
I think @maximilianoertel is right here. And so are you @Emily-ejag that this should apply to the export score and progress report tests as well. Let's strategize in the next sprint about how to do that. I think it might involve separating out the CSV download logic such that it can be called with mock data.